lib/rails/generators/appoptics_apm/templates/appoptics_apm_initializer.rb

```diff
@@ -61,6 +61,25 @@
   #
   AppOpticsAPM::Config[:tracing_mode] = :enabled
 
+  #
+  # Add traceId to logs
+  #
+  # Configure if and when the traceId should be included in application logs.
+  # Common Ruby and Rails loggers are auto-instrumented so that they can include
+  # the current traceId in log messages.
+  #
+  # The added string will look like: "ao.traceId=7435A9FE510AE4533414D425DADF4E180D2B4E36-0"
+  # It ends in '-1' if the request is sampled and in '-0' otherwise.
+  #
+  # The following options are available:
+  # :never (default)
```

Comment:
the `case` above also accepted `nil`. is that not a setting the user can set?

Reply:
It is also nil, if the user doesn't set anything. So, I think a specific word for the setting is preferrable, because it is clear that this is what the user wants and it is clear to the user what they are setting. It is a bit of a convention in the Ruby world.